### ISSUE TITLE
Memory leaks created by SpTickingWindowPresenter 

### ISSUE THREAD

**Issue by guillep** [pharo-spec/Spec#1127]
I inspected a large object using the basic inspector and that created a dangling reference in `SpNullApplication`.
Using reference finder I tracked that to SpTickingWindowPresenter

**Comment by Ducasse** [comment]
Thanks guille!

**Comment by estebanlm** [comment]
yeah, `SpTickingWindowPresenter` needs to be removed, in fact , no idea why I didn't do it before :)
